backend/src/main/resources/schema.graphqls

```diff
@@ -403,26 +414,26 @@ type Mutation {
     employedInHealthcare: Boolean!
   ): Patient @requiredPermissions(allOf: ["EDIT_PATIENT"])
   updatePatient(
-    facilityId: ID
-    patientId: ID!
+    facilityId: ID @Size(min: 36, max: 36)
+    patientId: ID! @Size(min: 36, max: 36)
     lookupId: String
     firstName: String!
     middleName: String
     lastName: String!
-    suffix: String
+    suffix: String @Size(min: 0, max: 32)
     birthDate: LocalDate!
     street: String!
     streetTwo: String
     city: String
-    state: String!
-    zipCode: String!
+    state: String! @Pattern(regexp: "[A-Z]{2}", message: "Use 2 letter state code")
+    zipCode: String! @Pattern(regexp: "[0-9]{5}(-[0-9]{4})?", message: "Use ZIP or ZIP+4 format")
```

Comment:
Not sure we fully support 5+4 ... I guess allowing it here isn't any worse than allowing anything.